### ISSUE TITLE
Currency ReferenceError

### ISSUE THREAD

**Issue by MxtOUT** [tfedor/AugmentedSteam#756]
https://github.com/tfedor/AugmentedSteam/issues/747#issuecomment-577951675

@andrewpros What info are you seeing if you expand the first ReferenceError?

**Comment by andrewpros** [comment]
https://i.imgur.com/DkFmW2u.png

But it is probably worth to mention i'm using kinda old firefox 56 due to irreplaceable extensions, but couldn't find any incompatible thing used, but it must be something with it as it works in 73.

For some reason _rates are undefined and it diesn't catch _rates assignment code

https://i.imgur.com/mz7r5yP.png

**Comment by HYBRID-BEING** [comment]
I've got the same problem on Waterfox 2020.01 (64-bit).
Probably not related, but 1.4.0 (i also had problems with that version, but console wasn't very informative about it) was not automatically updating to 1.4.1, so i had to reinstall it manually (without a hitch).

**Comment by andrewpros** [comment]
Ahh, thx for the info, i checked Mozilla extension website, and yeah, im stuck at 1.4.0 cuz 1.4.1 supports only fx 57+.
Hmm is waterfox a drop in replacement for old fx?

**Comment by andrewpros** [comment]
Well i managed to update at last my fx to the latest one and it works fine.
So it is up to you if u want to fix it for older fx.

**Comment by MxtOUT** [comment]
Yeah, just had the same report on Discord, older browsers aren't supported anymore (at least FF 57 is required)

**Comment by andrewpros** [comment]
Well i just didn't see what was the requirement for a fx 57 bump, but it was long overdue with my fx version and had to update it either way.

**Comment by HYBRID-BEING** [comment]
I might have worded these incorrectly, but i meant that script doesn't work on Waterfox 2020.01 even after updating to 1.4.1.
@andrewpros, yeah, Waterfox keeps NPAPI support that Firefox ditched, but it kinda lags behind in base version, i think (a bit hard to follow now, since they stating to call versions "year.month").

**Comment by MxtOUT** [comment]
I downloaded Waterfox 2020.01 and it seems like it's still based on FF 56...
![image](https://user-images.githubusercontent.com/14999077/74241326-13631c80-4cdc-11ea-87c8-4b84e217fd1f.png)
